server/src/main/java/com/flipkart/varadhi/utils/SubscriptionFactory.java

```diff
@@ -0,0 +1,33 @@
+package com.flipkart.varadhi.utils;
+
+import com.flipkart.varadhi.entities.SubscriptionResource;
+import com.flipkart.varadhi.entities.VaradhiSubscription;
+
+public final class SubscriptionFactory {
```

Comment:
Can we have these conversion methods in the base class itself, this seems redundant to me.

Reply:
also if it is still needed, Factory name is misleading.

Reply:
have changed to SubscriptionHelper